pom.xml

```diff
@@ -81,7 +81,9 @@
   <org.owasp.dependency.check.version>8.3.1</org.owasp.dependency.check.version>
   <com.google.cloud.tools.jib.maven.plugin.version>3.3.2</com.google.cloud.tools.jib.maven.plugin.version>
 
+  <!-- Use a separate property for dependency alignment purposes, keep the two versions in sync. -->
   <com.google.dagger.version>2.47</com.google.dagger.version>
+  <com.google.dagger.compiler.version>2.47</com.google.dagger.compiler.version>
```

Comment:
Is it possible to do `<com.google.dagger.compiler.version>${com.google.dagger.version}</com.google.dagger.compiler.version>`? And then whatever downstream process needs these separate should still be able to override one or the other with `-D`.

Reply:
I think I found a way to make that work.